extensions/wikia/GlobalNavigation/GlobalNavigation.i18n.php

```diff
@@ -442,6 +444,9 @@
 	'global-navigation-desc' => '{{desc}}',
 	'global-navigation-create-wiki' => 'Label on create new wikia button on Global Navigation - top bar',
 	'global-navigation-hubs-menu-more-of' => 'Global Navigation Hubs Menu - Link that goes to specific hub\'s mainpage. First parameter is hub\'s name (Video Games, Life, etc.)',
+	'global-navigation-register' => 'Link that shows registration modal',
+	'global-navigation-sign-in' => 'Link that shows sign-in modal',
+	'global-navigation-account-navigation-options' => 'Sentence that says to register or sign-in, $1 is "global-navigation-register", $2 is "global-navigation-sign-in"',
```

Comment:
I think we should inform translators that ```$1``` and ```$2``` are being wrapped by link and there is no need to put wikitext there.